composables/useCart.ts

```diff
@@ -1,9 +1,11 @@
+import { CartItemInput } from "./../server/types/GraphQL"
```

Comment:
instead of relative path use @ for consistency

Reply:
Done